eslint.config.mjs

```diff
@@ -0,0 +1,126 @@
+import mocha from 'eslint-plugin-mocha'
+import n from 'eslint-plugin-n'
+import globals from 'globals'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import js from '@eslint/js'
+import { FlatCompat } from '@eslint/eslintrc'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+})
+
+export default [
+  {
+    ignores: [
+      '**/coverage',
+      '**/dist',
+      '**/docs',
+      '**/out',
+      '**/node_modules',
+      '**/versions',
+      '**/acmeair-nodejs',
+      '**/vendor',
+      'integration-tests/esbuild/out.js',
+      'integration-tests/esbuild/aws-sdk-out.js',
+      'packages/dd-trace/src/appsec/blocked_templates.js'
+    ]
+  }, ...compat.extends('eslint:recommended', 'standard', 'plugin:mocha/recommended'), {
+    plugins: {
+      mocha,
+      n
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.node,
+        ...globals.mocha,
+        sinon: false,
+        expect: false,
+        proxyquire: false,
+        withVersions: false,
+        withPeerService: false,
+        withNamingSchema: false,
+        withExports: false
+      },
+
+      ecmaVersion: 2022
+    },
+
+    settings: {
+      node: {
+        version: '>=16.0.0'
+      }
+    },
+
+    rules: {
+      'max-len': [2, 120, 2],
+      'no-var': 2,
+      'no-console': 2,
+      'prefer-const': 2,
+      'object-curly-spacing': [2, 'always'],
+      'import/no-extraneous-dependencies': 2,
+      'standard/no-callback-literal': 0,
+      'no-prototype-builtins': 0,
+      'n/no-restricted-require': [2, ['diagnostics_channel']],
+      'n/no-callback-literal': 0,
+
+      'object-curly-newline': ['error', {
+        multiline: true,
+        consistent: true
+      }],
+
+      'import/no-absolute-path': 0,
+      'no-unused-expressions': 0
```

Comment:
I think it makes the file much more readable if you use the string literals for `off`, `warn`, and `error` instead of the numbers 0-2. So this block would be:

```suggestion
      'max-len': ['error', 120, 2],
      'no-var': 'error',
      'no-console': 'error',
      'prefer-const': 'error',
      'object-curly-spacing': ['error', 'always'],
      'import/no-extraneous-dependencies': 'error',
      'standard/no-callback-literal': 'off',
      'no-prototype-builtins': 'off',
      'n/no-restricted-require': ['error', ['diagnostics_channel']],
      'n/no-callback-literal': 'off',

      'object-curly-newline': ['error', {
        multiline: true,
        consistent: true
      }],

      'import/no-absolute-path': 'off',
      'no-unused-expressions': 'off'
```

And then the equivalent with the other `rules` objects below.

Reply:
@watson do you know of a plugin or rule that would enforce this?

Reply:
Haha I haven't checked if one exists, but I wouldn't be surprised

Reply:
Did some quick googling today and I haven't been able find anything 🙃